onecodex/viz/__init__.py

```diff
@@ -183,6 +183,9 @@ def find_spec(self, fullname, path, target=None):
         self.in_progress = True
         try:
             spec = importlib.util.find_spec(fullname)
+            # noop - altair is not installed
+            if spec is None:
+                return
```

Comment:
nit: this method should return a `spec` or `None`. `return None` would be clearer IMO